### PR TITLE
Core&Extension

### PR DESCRIPTION
**Pull request by LinusWillmont**

### CODE REVIEW

File: api-cinema-challenge/api-cinema-challenge/Repositories/booking/IBookingRepository.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using api_cinema_challenge.Models;
+
+namespace api_cinema_challenge.Repositories.customer
+{
+    public interface IBookingRepository
+    {
+        public Task<Booking?> Add(int customerId, int screeningId, int nrOfTickets);
```
Comment (dogezen) [inline api-cinema-challenge/api-cinema-challenge/Repositories/booking/IBookingRepository.cs:7]: naming could be improveD: ticketCount

Comment (dogezen) [reply api-cinema-challenge/api-cinema-challenge/Repositories/booking/IBookingRepository.cs:7]: or ticketQuantity

File: api-cinema-challenge/api-cinema-challenge/Repositories/booking/BookingRepo.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    {
+        CinemaContext _db;
+        public BookingRepo(CinemaContext db)
+        {
+            _db = db;
+        }
+
+        public async Task<Booking?> Add(int customerId, int screeningId, int nrOfTickets)
+        {
+            var screening = await _db.Screenings.FirstOrDefaultAsync(s => s.Id.Equals(screeningId));
```
Comment (dogezen) [inline api-cinema-challenge/api-cinema-challenge/Repositories/booking/BookingRepo.cs:17]: good error handling

**Review by dogezen — COMMENTED** [review]
very good, really not much to say

File: api-cinema-challenge/api-cinema-challenge/Controllers/BookingEndpoints.cs [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    public static class BookingEndpoints
+    {
+        public static void ConfigureBookingEndpoints(this WebApplication app)
+        {
+            var bookings = app.MapGroup("/bookings");
+
+            bookings.MapPost("", CreateBooking);
+            bookings.MapGet("", GetAllBookings);
+            app.MapGet("/customers/{customerId}/Bookings", GetAllBookingsByCustomer);
+            app.MapDelete("/{id}", DeleteBookingById);
```
[the remark this answers is not in the document]
Comment (LinusWillmont) [reply api-cinema-challenge/api-cinema-challenge/Controllers/BookingEndpoints.cs:17]: Not sure how to approach these URLs. 
I'm thinking that placing the GetBookingsByCustomer method under the customer makes sense.
However, I made a mistake with the DeleteBookingById method, that one should definitely be inside bookings.

**Pull request closed without merging**